packages/@guardian/source-react-components-development-kitchen/src/components/toggle-switch/styles.ts

```diff
@@ -25,13 +30,12 @@ export const toggleSwitchStyles = css`
 `;
 
 export const mediumStyles = css`
-	button {
-		width: 3.188rem;
-		height: 1.938rem;
-		border-radius: 15.5px;
-	}
+	${buttonStyles}
```

Comment:
You could also just include `buttonStyles` in the main element, as in: `css={[buttonStyles, isSlim ? slimStyles : mediumStyles]}`

This is perhaps a bit easier to reason about because then adding slimStyles is _only_ adding that css

Reply:
Definitely looks better this way, thanks 👍